setup.cfg

```diff
@@ -5,12 +5,15 @@ include_trailing_comma = True
 known_first_party = transformers
 known_third_party =
     absl
+    elasticsearch
```

Comment:
@yjernite New packages for the example folder have to be added here to avoid problems with isort (learned from @sshleifer)